codice/ddf-ui
# Added security for workspaces.

#### What does this PR do?

Adds role based security to workspaces.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@pklinef 
@stustison 
@jlcsmith 
@kcwire 
@rzwiefel 
@andrewkfiedler 
#### How should this be tested?
1. Create a workspace as a user (guest cannot create workspaces).
2. See that no other user can initially access the workspace.
3. Share a work with another user via a role.
4. See that the other user can now see the workspace.
#### What are the relevant tickets?

[DDFUI-4](https://codice.atlassian.net/browse/DDFUI-4)
#### Checklist:
